tests/e2e/dsc_deletion_test.go

```diff
@@ -123,3 +133,24 @@ func (tc *testContext) testAllApplicationDeletion() error {
 	}
 	return err
 }
+
+func (tc *testContext) testDSCIDeletion() error {
```

Comment:
Basically, it's not test but just cleanup, isn't it?

Reply:
it is a similar one to `testDSCDeletion` 
it was in the "ConfigMap deletion" test suite, now, moved into "deletion" test suite. 
both two functions are (could be a misleading function name) looking for named DSC or DSCI CR and delete it. 
based on the pre-condition, CR should exist in the cluster, so we do not expect error.
But if CR is not in the cluster (not found) or error during .Delete() , we fail test

Reply:
i renamed function to `testDeletionExistDSCI()`
does it make more sense?

Reply:
> CR should exist in the cluster, so we do not expect error.

Reply:
added comments in function